apps/design-system-docs/docs/foundation/datavisualisation.md

```diff
@@ -1,10 +1,9 @@
 # Data Visualisation
-
 A graphical presentation of information in a way that makes it easier to understand and process. It is usually interactive and allows for the user to draw their own conclusions about the data.
 
 ## Infographics
 
-An infographic is very similar to a data visualisation since it too is a graphical presentation of information that is presented in a way that is easy to digest and understand. Infographics are more subjective though by telling a complete and premeditated story which guides the user to a particular conclusion. Refer to the Equinor  for guidelines on creating infographics.
+An infographic is very similar to a data visualisation since it too is a graphical presentation of information that is presented in a way that is easy to digest and understand. Infographics are more subjective though by telling a complete and premeditated story which guides the user to a particular conclusion. Refer to the [Equinor Brand Center](../resources/resources.md#resources) for guidelines on creating infographics.
```

Comment:
```suggestion
An infographic is very similar to a data visualisation since it too is a graphical presentation of information that is presented in a way that is easy to digest and understand. Infographics are more subjective though by telling a complete and premeditated story which guides the user to a particular conclusion. Refer to the [Equinor Brand Center](https://communicationtoolbox.equinor.com/point/en/equinor/) for guidelines on creating infographics.
```